src/python/WMComponent/WorkflowUpdater/WorkflowUpdaterPoller.py

```diff
@@ -65,6 +226,15 @@ def __init__(self, config):
                            hostUrl=self.rucioUrl,
                            authUrl=self.rucioAuthUrl)
                            # configDict={'logger': self.logger})
+        # define where to look for sandbox tar balls
+        self.idir = getattr(self.config.WorkflowUpdater, "componentDir")
```

Comment:
I prefer to make it as separate issue. The provided changes in this PR address quite complex issue and brining additional changes may impact different parts and affect deployment. My view on this is to first verify that logic we put in this PR works as expected and then address changes in configuration (which will require changes in existing WMA).

Reply:
On this we have different opinions. If we have 1 ticket asking for a feature, I would rather have 1 PR providing that feature instead of splitting developments (dependencies) across multiple contributions. 
This will be 1 additional line to the configuration file.

Reply:
ok, changed to 
```
self.idir = getattr(self.config.WorkflowUpdater, "sandboxDir")
```
and removed check for directory existance.